releasenotes/notes/default-schedule-name-51ba198cf08978cd.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixes a case in Result.get_counts(), where the results for an expirement
+    could not be referenced if the experiment was initialized as a Schedule without a name.
+    Refer to `#2753 <https://github.com/Qiskit/qiskit-terra/issues/2753>` for more details.
```

Not a blocker, but the link probably won't render correctly just need to add a missing character:

```suggestion
    Refer to `#2753 <https://github.com/Qiskit/qiskit-terra/issues/2753>`_ for more details.
```